src/electron/views/renderer-initializer.ts

```diff
@@ -264,6 +271,9 @@ getPersistedData(indexedDBInstance, indexedDBDataKeysToFetch).then(
         );
         featureFlagStore.initialize();
 
+        const leftNavStore = new LeftNavStore(leftNavActions);
+        leftNavStore.initialize({ selectedKey: 'automated-checks' });
```

Comment:
I think you can call initialize without any parameters here. The default state that you've dictated within the store will be used instead.